### ISSUE TITLE
Error reverting: Remove file `/Library/LaunchDaemons/org.nixos.darwin-store.plist`

### ISSUE THREAD

**Issue by MarkusZoppelt** [DeterminateSystems/nix-installer#1356]
## Error
```
Error: 
   0: Error reverting
   0: Action `create_volume_service` errored
      Remove file `/Library/LaunchDaemons/org.nixos.darwin-store.plist`

```

## Metadata
|key|value|
|--|--|
|**version**|0.27.1|
|**os**|macos|
|**arch**|aarch64|

**Comment by cole-h** [comment]
Do you have any more information on how this happened?

Did you install, fail to install, and then accept the proposed revert and run into this? Or were you just uninstalling because you wanted to?

**Comment by MarkusZoppelt** [comment]
It removed pretty much everything except these files, which I manually removed with sudo:

```
/Library/LaunchDaemons/.before-determinate-nixd.org.nixos.*
/Library/LaunchDaemons/org.nixos.nix-daemon.plist
/Library/LaunchDaemons/systems.determinate.nix-store.plist
```

**Comment by cole-h** [comment]
Ah, OK, that makes sense to me -- IIRC 0.27.1 was the last version before we fixed some logic bugs around cleanup of files like that. Thanks for the info and bug report! I'm gonna close this, since I believe this class of issue has been fixed in 0.28.0 and later versions.
